tests/Tests/ORM/Mapping/XmlMappingDriverTest.php

```diff
@@ -310,6 +311,39 @@ public function testDisablingXmlValidationIsPossible(): void
             false,
         );
     }
+
+    public function testXmlValidationEnabled(): void
+    {
+        self::expectException(MappingException::class);
+        self::expectExceptionMessage("libxml error: Element '{http://doctrine-project.org/schemas/orm/doctrine-mapping}field', attribute 'invalid': The attribute 'invalid' is not allowed.");
```

Comment:
Please move both lines right before the line supposed to trigger them, it's a best practice.

Reply:
Moved